ComponentKit/Utilities/CKArrayControllerChangesetVerification.h

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <Foundation/Foundation.h>
+
+#import <ComponentKit/CKArrayControllerChangeset.h>
+
+/**
+ This function determines whether a given changeset is valid for a particular data source state
```

Comment:
It may be nice to mention what exactly this function considers to be a valid changeset.